### PR TITLE
Hot fix provider choices in list filter forms

### PR DESCRIPTION
**Pull request by juggler31**
## Description

Update provider choice call with tests

## Checklist

- [ ] I have followed the guidelines in our [Contributing document](https://geotrek.readthedocs.io/en/latest/CONTRIBUTING.html)
- [ ] My code respects the Definition of done available in the [Development section of the documentation]()https://geotrek.readthedocs.io/en/latest/contribute/development.html#definition-of-done-for-new-model-fields
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes
- [ ] I added an entry in the changelog file
- [ ] My commits are all using prefix convention (emoji + tag name) and references associated issues
- [ ] I added a label to the PR corresponding to the perimeter of my contribution
- [ ] The title of my PR mentionned the issue associated

### CODE REVIEW

File: geotrek/signage/tests/test_views.py [hunk 1]
```diff
@@ -14,6 +14,7 @@
 from geotrek.authent.tests.factories import PathManagerFactory, StructureFactory
 from geotrek.common.tests.factories import OrganismFactory
 from geotrek.signage.models import Signage, Blade
+from geotrek.signage.views import SignageList
```
Comment (babastienne) [inline geotrek/signage/tests/test_views.py:17]:
```suggestion
```

File: geotrek/signage/managers.py [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
         return tuple((value, value) for value in values)
 
-    def provider_choices(self):
-        providers = self.get_queryset().existing().exclude(provider__exact='') \
-            .distinct('provider').values_list('provider', 'provider')
-        return providers
+    # def provider_choices(self):
+    #     providers = self.get_queryset().existing().exclude(provider__exact='') \
+    #         .distinct('provider').values_list('provider', 'provider')
+    #     return providers
```
Comment (submarcos) [inline geotrek/signage/managers.py:15]: delete this comment part

**Review by submarcos — COMMENTED** [review]
comments to delete

File: geotrek/trekking/tests/test_views.py [hunk 3]
```diff
@@ -1653,3 +1654,78 @@ def test_delete_infrastructure_refreshes_pdf(self, mock_get):
...
+    def test_provider_filter_without_provider(self):
+        filter_set = ServiceFilterSet(data={})
+        filter_form = filter_set.form
+
+        self.assertTrue(filter_form.is_valid())
+        self.assertEqual(0, filter_set.qs.count())
+
+    def test_provider_filter_with_providers(self):
+        signage1 = ServiceFactory.create(provider='my_provider1')
+        signage2 = ServiceFactory.create(provider='my_provider2')
```
Comment (Chatewgne) [inline geotrek/trekking/tests/test_views.py:1722]: Should be `service1` and `service2`

File: geotrek/trekking/tests/test_views.py [hunk 4]
```diff
@@ -1653,3 +1654,78 @@ def test_delete_infrastructure_refreshes_pdf(self, mock_get):
...
+    def test_provider_filter_without_provider(self):
+        filter_set = POIFilterSet(data={})
+        filter_form = filter_set.form
+
+        self.assertTrue(filter_form.is_valid())
+        self.assertEqual(0, filter_set.qs.count())
+
+    def test_provider_filter_with_providers(self):
+        signage1 = POIFactory.create(provider='my_provider1')
+        signage2 = POIFactory.create(provider='my_provider2')
```
Comment (Chatewgne) [inline geotrek/trekking/tests/test_views.py:1697]: Should be `poi1` and `poi2`

**Review by Chatewgne — CHANGES_REQUESTED** [review]

**Review by submarcos — APPROVED** [review]

**Pull request merged**